### PR TITLE
fixed enterVehicle()

### PR DESCRIPTION
**Pull request by redv**
- fixed issue #253 "Now any number of agents can be added to a craft, regardless of number of passenger modules!"
- pixelperfection for AgentAssignment

### CODE REVIEW

File: data/forms/city/agentassignment.form [hunk 1]
```diff
@@ -34,21 +34,13 @@
         <imagedepressed>BUTTON_SCROLL_DOWN_DEPRESSED</imagedepressed>
     </graphicbutton>
     <multilistbox id="AGENT_SELECT_BOX" scrollbarid="AGENT_SELECT_SCROLL">
-        <position x="16" y="32"/>
```
Comment (JonnyH) [inline data/forms/city/agentassignment.form]: Was this change intended to be in this commit?

If it's fixing a different issue (e.g. misaligned form), it probably makes sense to have a separate commit

Comment (makus82) [reply data/forms/city/agentassignment.form]: Hey Jonny offtopic but look pls i create PR wich readme)
